### PR TITLE
Added typescript typings

### PR DESCRIPTION
**Pull request by tomitrescak**

### CODE REVIEW

File: lib/storybook.d.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+declare var module: any; // dangerous
```
Comment (arunoda) [inline lib/storybook.d.ts]: Could you rename this `lib` directory to `config`.

**Comment by tomitrescak** [comment]
I have renamed lib to config.

**Comment by arunoda** [comment]
Great.

File: config/storybook.d.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+declare var module: any; // dangerous
+
+declare module "@kadira/storybook" {
+  interface Story {
+      add (storyName: string, callback: Function): Story;
+  }
+
+  export function storiesOf(name: string, module: any): Story;
+  export function action(name: string, ...params: any[]): Function;
```
Comment (arunoda) [inline config/storybook.d.ts]: There's another API called [linkTo](https://github.com/kadirahq/react-storybook/blob/master/src/client/client_api.js#L67). Add that too.

**Comment by arunoda** [comment]
We are currently working on a major refactor at the `mantra` branch. We are about to release it by next couple of days.
I'll take this with that.

**Pull request merged**